lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```
@@ -241,7 +243,16 @@ server.information.qgis.name=Name
 server.information.qgis.plugins=Plugins
 server.information.qgis.plugins.version=Version number
 
-server.information.qgis.test.ok=QGIS Server is correctly installed and returns the expected response.
-server.information.qgis.test.error=QGIS Server is not correctly installed in your server or the URL given in Lizmap configuration is not correct.
+server.information.qgis.test.ok=QGIS Server is correctly installed and returns the expected response for WMS requests.
```

```suggestion
server.information.qgis.test.ok=QGIS Server is correctly installed and returns the expected response for OGC requests.
```

It's not specific to WMS requests.